.github/workflows/snapshot-release.yml

```diff
@@ -56,6 +56,11 @@ jobs:
         with:
           ref: ${{ steps.refs.outputs.head_ref }}
 
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.19
```

Comment:
Is there a way to tell the action to use the version of `go.mod`?

Reply:
Yes, there is: https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file